Plots/Contours/NCL_color_1.py

```diff
@@ -0,0 +1,109 @@
+"""
+NCL_color_1.py
+===============
+This script illustrates the following concepts:
+   - Drawing a horizonal color bar
+   - Adjusting a colorbar position relative to plot axes
+   - Recreating a default NCL colormap
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/color_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/color_1_lg.png
+"""
+
+###############################################################################
+# Import packages:
+
+import numpy as np
+import matplotlib.pyplot as plt
+import xarray as xr
+import cartopy.crs as ccrs
+
+import geocat.datafiles as gdf
+from geocat.viz import cmaps as gvcmaps
+import geocat.viz.util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarray
+ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc")).isel(time=1)
+
+###############################################################################
+# Plot:
+
+# Generate figure and set its size in (width, height)
+fig = plt.figure(figsize=(10, 8))
+
+# Generate axes using Cartopy to draw coastlines
+ax = plt.axes(projection=ccrs.PlateCarree())
+ax.coastlines(linewidth=0.5, alpha=0.6)
+
+# Use geocat.viz.util convenience function to set axes limits & tick values
+gvutil.set_axes_limits_and_ticks(ax,
+                                 xlim=(-180, 180),
+                                 ylim=(-90, 90),
+                                 xticks=np.linspace(-180, 180, 13),
+                                 yticks=np.linspace(-90, 90, 7))
+
+# Use geocat.viz.util convenience function to add minor and major tick lines
+gvutil.add_major_minor_ticks(ax, labelsize=10)
+
+# Use geocat.viz.util convenience function to make latitude, longitude tick labels
+gvutil.add_lat_lon_ticklabels(ax)
+
+# Import the default color map
+# Note: this may not be the best colormap to interpret the information, but was
```

Comment:
Could you move this important `Note` comment to the upper function docstring so that the user could initially see this?

Reply:
Added it, thanks for the suggestion!